xikolo-common/Data/Model/Channel.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Channel.swift
+//  xikolo-ios
+//
+//  Created by Bjarne Sievers on 27.10.16.
+//  Copyright © 2016 HPI. All rights reserved.
+//
+
+import Foundation
+import CoreData
+import Spine
+import UIKit
+
+class Channel: BaseModel {
+
+    /*var color: UIColor {
+        return CGColor.
+    }*/
```

Comment:
I forgot about that one. Would this be an appropriate place to do this? Should it return a CGColor or UIColor object?